solidity/test/bridge/Bridge.test.ts

```diff
@@ -1493,164 +1493,323 @@ describe("Bridge", () => {
 
         context("when main UTXO data are valid", () => {
           context("when redeemer output script is standard type", () => {
+            // Arbitrary standard output scripts.
+            const redeemerOutputScriptP2WPKH =
+              "0x160014f4eedc8f40d4b8e30771f792b065ebec0abaddef"
+            const redeemerOutputScriptP2WSH =
+              "0x220020ef0b4d985752aa5ef6243e4c6f6bebc2a007e7d671ef27d4b1d0db8dcc93bc1c"
+            const redeemerOutputScriptP2PKH =
+              "0x1976a914f4eedc8f40d4b8e30771f792b065ebec0abaddef88ac"
+            const redeemerOutputScriptP2SH =
+              "0x17a914f4eedc8f40d4b8e30771f792b065ebec0abaddef87"
+
             context(
               "when redeemer output script does not point to the wallet public key hash",
               () => {
                 context("when amount is not below the dust threshold", () => {
```

Comment:
s/not below/above ?

Reply:
Not quite. The "not below" is used because it should be greater or equal.